polaris-react/src/components/OptionList/tests/OptionList.test.tsx

```diff
@@ -539,12 +539,9 @@ function firstOption(
   options?: OptionDescriptor[],
   sections?: OptionListProps['sections'],
 ): string {
-  const firstOptionsValue =
-    options == null || options === [] ? '' : options[0].value;
+  const firstOptionsValue = options == null ? '' : options[0].value;
   const firstSectionOptionsValue =
-    sections == null || sections === [] || sections[0].options === []
-      ? ''
-      : sections[0].options[0].value;
+    sections == null ? '' : sections[0].options[0].value;
```

Comment:
The array checks were unnecessary with the null check. Removed them and tests function the same.